### DISCUSSION TITLE
Short detail display of the user's collection?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#13065] [Archive]
To `directus_users` collection, I added an M2O field where the user has to express a choice.
Other part of the app will behave according to his choice.

ATM the M2O choice field is at the bottom of its (long) detail page.

Apart the full detail page he has to see when clicking on his avatar, 
Is there a way to show a "short version" of his detail page: first_name and the choice M2O field?
or, even better, show directly the selection (the one that appears on the right panel of user's detail).

**Comment by rijkvanzanten** [comment]
There's currently no way to reconfigure the layout of the user detail page (beyond limiting read access to certain fields, which makes them disappear)
